rst2pdf/style-to-yaml.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+'''
+    Convert older RSON stylesheets to YAML format
+
+    Run the script with the filename to convert, it outputs to stdout
+'''
+
+import sys
+import json
+import yaml
+from rst2pdf.dumpstyle import fixstyle
+
+from rson import loads as rloads
+
+if __name__ == '__main__':
+    fname = sys.argv[1]
```

Comment:
Rather than parsing `argv` directly like this, could we use `argparse`? It's trivial and would allow people to get information on expected input via e.g. `-h`. Also, it would make it easier to convert multiple files at once.

```python3
#!/usr/bin/env python3

"""Convert older RSON stylesheets to YAML format.

Convert the file at PATH from RSON to YAML format.
"""

import argparse
import json
import sys

import yaml

from rst2pdf.dumpstyle import fixstyle

def main():
    parser = argparse.ArgumentParser(
        description=__doc__,
        formatter_class=argparse.RawDescriptionHelpFormatter,
    )
    parser.add_argument(
        'path',
        dest='paths',
        metavar='PATH',
        nargs='+',
        help='An RSON-formatted file to convert.',
    )
    args = parser.parse_args()
    for path in args.paths:
        pass # stuff here...
```

Reply:
oh this is so much nicer now we've turned it more into a CLI thing, and I learned something too!